### PR TITLE
Fixes #DF-1732 Blog/Newsroom Sidebar Update

### PR DESCRIPTION
**Pull request by sebworks**
Update Newsroom and Blog post sidebars

## Changes
 
- Updated ' _includes/popular-stories.html' to add description and date.
- Updated ' _includes/related-posts.html'  to add description and date.
- Updated ' newsroom/index.html' to add class.
- Updated ' static/css/post.less' to change link style in sidebar.

## Review
 
- @jimmynotjim  
- @anselmbradford 

## Preview
 
screen shot 2015-03-30 at 12 17 27 pm

### CODE REVIEW

File: static/css/post.less [hunk 1]
```diff
@@ -457,15 +457,24 @@
 
     .list_link {
         .h4();
-        .u-link__colors(@black, @pacific-80);
-        .u-link__no-border();
+        .u-link__colors(@pacific, @pacific, @pacific-50, @pacific, @navy);
+        border-width: 0;
+
+        &:hover {
+            border-bottom: 1px dotted;
```
Comment (anselmbradford) [inline static/css/post.less]: Is there any problem with not having the color defined here? I'm assuming the border is defined elsewhere and this is overriding the width and style?

File: static/css/post.less [hunk 2]
```diff
@@ -457,15 +457,24 @@
...
+        }
 
         .respond-to-min(600px, {
             .h3();
+            font-size: unit(16px / @base-font-size-px, em);;
```
Comment (anselmbradford) [inline static/css/post.less]: Minor... extra trailing `;`.

File: static/css/post.less (hunk 1 above)
Comment (Scotchester) [inline static/css/post.less]: Correct; they're set in the cf-core link styles.

Comment (anselmbradford) [inline static/css/post.less]: :+1:

**Comment by anselmbradford** [comment]
:+1: Just may want to remove https://github.com/cfpb/cfgov-refresh/pull/441#discussion_r27418575, squash and merge

**Pull request merged**